README.md

```diff
@@ -1,2 +1,8 @@
 # Axe-Puppeteer-and-HTML
-Example of using Puppeteer to run accessibility checks on HTML code
+This repo is an example of using Puppeteer to run accessibility checks on HTML code. The main script that is run is called `a11y-check.js`
+
+Essentially, Puppeteer runs in headless mode, and the axe-core package code is injected into the "browser" as script. If axe finds any serious or critical issues, it will output the array of violations into the console.
```

Comment:
```suggestion
Puppeteer is a browser automation tool that can be run in headless mode, we can then inject the axe-core package into the headless browser as a script. If Axe finds any serious or critical issues, it will output the array of violations into the console.
```